src/Inputs/RadioButtons.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Jeremeamia\Slack\BlockKit\Inputs;
+
+use Jeremeamia\Slack\BlockKit\Exception;
+use Jeremeamia\Slack\BlockKit\Partials\Option;
+
+class RadioButtons extends InputElement
+{
+
+    use HasConfirm;
+
+    private const MIN_OPTIONS = 1;
+    private const MAX_OPTIONS = 10;
+
+    /** @var array|Option[] */
+    private $options = [];
+
+    /** @var Option|null */
+    private $initialOption = null;
+
+    public function addOption(Option $option, bool $isInitial = false): RadioButtons {
```

Comment:
You can remove this method and rely on `HasOptions:option()`/`HasOptions::options()` from the trait.